spec/acceptance/broker_spec.rb

```diff
@@ -254,6 +254,7 @@ class { 'kafka::broker':
         it { is_expected.to contain "Environment='KAFKA_LOG4J_OPTS=-Dlog4j.configuration=file:/tmp/log4j.properties'" }
         it { is_expected.to contain "Environment='KAFKA_OPTS=-Djava.security.policy=/some/path/my.policy'" }
         it { is_expected.to contain "Environment='LOG_DIR=/some/path/to/logs'" }
+        it { is_expected.to contain "LimitNOFILE='65536'"}
```

Comment:
I'm not sure the acceptance tests are being run automatically by travis.  Do you think you could also add an rspec-puppet test to https://github.com/voxpupuli/puppet-kafka/blob/master/spec/classes/broker_spec.rb ?